### PR TITLE
TH-883 | New routes/paths for selected places

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Add new routes for selected places (annantalo etc.)

## Issues :bug:
**[TH-883](https://helsinkisolutionoffice.atlassian.net/browse/TH-883):** 

## Screenshots :camera_flash:
Screenshot 2020-12-02 at 18 14 03

### CODE REVIEW

File: src/domain/eventSearch/constants.ts [hunk 1]
```diff
@@ -112,3 +112,13 @@ export const MAPPED_CATEGORIES: Record<string, string> = {
   [CATEGORIES.SPORT]: SPORT_KEYWORDS.join(','),
   [CATEGORIES.THEATRE]: 'yso:p2625',
 };
+
+export const MAPPED_PLACES: Record<string, string> = {
+  annantalo: 'tprek:7254',
```
Comment (nenonja) [inline src/domain/eventSearch/constants.ts:117]: Where are these codes coming from `tprek:7256`. Could they change? Are these codes the same in every environment (test/production)?

Comment (Sartonon) [reply src/domain/eventSearch/constants.ts:117]: https://api.hel.fi/linkedevents/v1/place/?division=kunta:helsinki&text=annantalo
https://api.hel.fi/linkedevents-test/v1/place/?division=kunta:helsinki&text=annantalo

They are id:s for the places. I figured them out just by searching the places from api with text. At leas Annantalo seems to be same id in both environments

Comment (Sartonon) [reply src/domain/eventSearch/constants.ts:117]: Would be strange if those ids change

Comment (jorilindell) [reply src/domain/eventSearch/constants.ts:117]: Those place ids are imported to Linked Events/Courses from tapahtumapaikkarekisteri and are same for all the environments. So this should be ok.

**Review by jorilindell — APPROVED** [review]
Looks good to me

**Pull request merged**